easybuild/easyconfigs/i/ImageJ/ImageJ-1.52q.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'PackedBinary'
+
+name = 'ImageJ'
+version = '1.52q'
```

Comment:
I think we should have a versionsuffix for the java version.

Reply:
@JackPerdue any update on this one?